com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/cf/clients/ServiceWithAlternativesCreatorTest.java

```diff
@@ -31,7 +34,7 @@
     private static final String CREATE_SERVICE_URL = "/v2/service_instances?accepts_incomplete=true";
     private static final String SPACE_ID = "TEST_SPACE";
 
-    @Parameters
+    @Parameters(name="{0}")
```

Comment:
Why name="{0}"?

Reply:
That way the in the IDE the test's names are going to be shown as the JSON file's names. Now we are going to know what is testCase7 failed.